src/main/java/fr/insalyon/creatis/moteurlite/runner/JobSubmitter.java

```diff
@@ -70,7 +70,8 @@ private void createJobs() throws MoteurLiteException {
                 }
 
                 String invocationString = convertMapToJson(finalInvocationInputs, boutiquesInputs);
-                String jobId = applicationName + "-" + System.nanoTime() + ".sh";
+                // jobId can be used for filenames, so normalize it by removing spaces
+                String jobId = applicationName.replace(' ', '_') + "-" + System.nanoTime() + ".sh";
```

Comment:
Could it be necessary to verify that spaces in the `applicationName` given in the 2 first parameters of the `GaswInput` parameter just bellow have no impact ?
Do you want me to check ?